plugin.video.montreal.greek-tv/addon.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.montreal.greek-tv" name="Montreal Greek TV" version="4.0.0" provider-name="Greektimes.ca">
+    <requires>
+		<import addon="xbmc.python" version="3.0.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>video audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <summary lang="en_GB">Montreal Greek TV addon</summary>
+        <description lang="en_GB">Montreal Greek TV add-on for Kodi featuring Live TV, Radio and Video on demand streams from Montreal, Quebec, Canada. Montreal Greek Television is broadcasting in-house locally-produced Canadian programming in the Greek language from the Greek Community of Greater Montreal.</description>
+        <summary lang="el_GR">Πρόσθετο Montreal Greek TV</summary>
+        <description lang="el_GR">Το πρόσθετο Montreal Greek TV για το σύστημα πολυμέσων Kodi προσφέρει ζωντανή τηλεόραση, ραδιόφωνο και ροές κατ' απαίτηση από το Μόντρεαλ, Κεμπέκ, Καναδά. Η Ελληνική Τηλεόραση του Μόντρεαλ (Montreal Greek TV) εκπέμπει περιεχόμενο τοπικής παραγωγής εις την Ελληνική γλώσσα από την Ελληνική παροικία της ευρύτερης περιοχής του Μόντρεαλ.</description>
+        <language>en el</language>
+        <license>GNU General Public License, v3</license>
```

Comment:
Please use a spdx compliant identifier for the license. Should be `GPL-3.0-only` or `GPL-3.0-or-later` depending on how you license your addon

Reply:
resolved